posts/2022-02-08-Rust-Survey-2021.md

```diff
@@ -0,0 +1,65 @@
+
+Adopting Rust at work seems to follow a challenging but ultimately worthwhile path for a lot of Rustaceans. First, 83% of those who have adopted Rust at work found it to be "challenging". How much this is related to Rust itself versus general challenges with adopting a new programming language, however, is unclear. During adoption only 13% of respondents found the language was slowing their team down; which contrasts with the 82% who found that Rust helped their teams achieve their goals.
+
+After adoption, the costs seem to be worth it: only 1% of respondents did not find the challenge worth it while 79% said it definitely was. When asked if their teams were likely to use Rust again in the future, 90% agreed.
+
+![Adopting Rust](/images/2022-02-RustSurvey/rust-adoption.png)
+
+Overall, Rust seems to be a language ready for the challenges of production, with only 3% of respondents saying that Rust was a "risky" choice for production use.
+
+## Challenges Ahead
+
+Overall, the annual survey points towards a growing, healthy community of Rustaceans, but this is not to say we don't have work ahead of us. Compile times, a historical focus of improvement for the Rust project, continue to not be where they need to be, with 61% of respondents saying work still needs to be done to improve them. Although, to the compiler team's credit, 61% found that they improved over the past year.  Other areas indicated as in need of more improvement were disk space (45%), debugging (40%), and GUI development (56%). 
+
+The IDE experience (led through continued adoption and improvement of rust-analyzer) gets the prize for showing the most improvement:  56% found it has improved over the last year.
+
+However, compiler error messages received the most praise, with 90% approval of their current state. 🎉
+
+When asked what their biggest worries for the future of Rust are, the top answer was a fear that there will not be enough usage in industry (38%). Given that Rust continues to show strides in adoption at places of work, the community seems to be on a good path to overcoming this concern. 
+
+The next largest concern was that the language would become too complex (33%). This was combined with a relative small number of folks calling for additional features (especially for ones not already in the pipeline). 
+
+Finally, the third largest concern was that those working on Rust would not find the proper support they need to continue to develop the language and community in a healthy way. With the establishment of the Rust Foundation, support structures are coming into place that hopefully will address this point, but no doubt plenty of work is still ahead of us.
+
+## An Exciting Future
+
+2021 was arguably one of the most significant year's in Rust's history - with the establishment of the Rust foundation, the 2021 edition, and a larger community than ever, Rust seems to be a solid path as we head into the future. 
```

Comment:
```suggestion
2021 was arguably one of the most significant year's in Rust's history - with the establishment of the Rust foundation, the 2021 edition, and a larger community than ever, Rust seems to be on a solid path as we head into the future. 
```

Reply:
Also it should be "most significant years in", no apostrophe.